api/docs/tools.md

```diff
@@ -86,31 +88,37 @@ def authority_options(cmd_name, args)
   op.add_option(
       "--authority [AUTHORITY,...]",
       ->(opts, v) { opts.authority = v},
-      "Comma-separated list of user authorities to add or remove for the users. ")
+      "Comma-separated list of user authorities to add or remove for the users. " +
+      "When granting authorities, use DEVELOPER to gain full access. " +
+      "Exactly one of --authority or --remove-all must be passed.")
   op.add_option(
       "--remove",
       ->(opts, _) { opts.remove = "true"},
-      "Remove authorities (rather than adding them.)")
+      "Remove authorities (rather than adding them) when using the --authority argument.")
+  op.add_option(
+    "--remove-all",
+    ->(opts, _) { opts.remove_all = true},
+    "Removes all authorities from user(s). Exactly one of --authority or --remove-all must be passed."
+  )
   op.add_option(
       "--dry_run",
       ->(opts, _) { opts.dry_run = "true"},
       "Make no changes.")
-  op.add_validator ->(opts) { raise ArgumentError unless opts.email and opts.authority}
+  op.add_validator ->(opts) { raise ArgumentError unless opts.email and ((opts.authority != "") ^ opts.remove_all)}
```

Comment:
why did you need to add the `!= ""` here? naively, the existing presence check looked fine

Reply:
The XOR operator doesn't auto-cast strings (`undefined method `^' for "":String (NoMethodError)`) and I was running into errors trying to cast `authority` directly to a boolean like before. And since I was trying to remove our Ruby code if possible, I wasn't particularly interested in researching the exact ruby-ism to describe this.

I'm merging this as-is but I'm open to any suggestions for a follow-up.